ibis/backends/risingwave/__init__.py

```diff
@@ -131,22 +155,37 @@ def create_table(
         overwrite
             If `True`, replace the table if it already exists, otherwise fail
             if the table exists
-
+        connector_properties
+            The properties of the sink connector, providing the connector settings to push to the downstream data sink.
+            Refer https://docs.risingwave.com/docs/current/data-delivery/ for the required properties of different data sink.
+        data_format
+            The data format for the new source, e.g., "PLAIN". data_format and encode_format must be specified at the same time.
+        encode_format
+            The encode format for the new source, e.g., "JSON". data_format and encode_format must be specified at the same time.
+        encode_properties
+            The properties of encode format, providing information like schema registry url. Refer https://docs.risingwave.com/docs/current/sql-create-source/ for more details.
+
+        Returns
+        -------
+        Table
+            Table expression
         """
         if obj is None and schema is None:
             raise ValueError("Either `obj` or `schema` must be specified")
 
-        if database is not None and database != self.current_database:
-            raise com.UnsupportedOperationError(
-                f"Creating tables in other databases is not supported by {self.name}"
+        if connector_properties is not None and (
+            encode_format is None or data_format is None
+        ):
+            raise com.RelationError(
```

Comment:
```suggestion
            raise com.UnsupportedArgumentError(
```

I think this is more appropriate here, but not a blocker

Reply:
Updated. But kind of confused: what's the differences between `RelationError `, `UnsupportedArgumentError` and `ValueError`?

Reply:
It's confusing!

`com.RelationError` should maybe go away, but currently it's only for internal errors related to expression construction -- no user should ever see a `RelationError`.

`ValueError` is (in my opinion) generally too vague to be useful, but it's tricky to balance adding a bunch of custom exceptions.

`UnsupportedArgumentError` seemed like a good fit to me, but there isn't a strict guide on when to use which exceptions in Ibis.  I think it's something we can improve for new contributors.

Reply:
OKK, thanks for the explanation!
As there's no comments left, can we merge it now?